src/reader/DifficultyFeedbackBox.js

```diff
@@ -12,25 +11,29 @@ import SentimentNeutralOutlinedIcon from '@mui/icons-material/SentimentNeutralOu
 
 let FEEDBACK_OPTIONS = { "Easy": 1, "Ok": 3, "Difficult": 5 };
 
-export default function DifficultyFeedbackBox({ api, articleID }) {
-  const [answerSubmitted, setAnswerSubmitted] = useState(false);
+export default function DifficultyFeedbackBox({ api, articleID, articleInfo, setArticleInfo, setAnswerSubmitted }) {
   const [isHovered, setIsHovered] = useState(false);
 
   function submitAnswer(answer) {
```

Comment:
rename to `updateArticleDifficultyFeedback`, move to `ArticleReader`, and pass a reference to it here. WIll simplify this component a lot, and you won't need all the params anymore.

Reply:
ArticleReader is only present in web, and is equivalent to ReadArticle in the browser-extension. If we moved the function to ArticleReader, we would need to have a similar function in ReadArticle. We think that this does not make sense, because they would be similar functions. Do you agree?

A solution could be to make a component for the entire feedback section, which returned both the LikeFeedbackBox and the DifficultyFeedbackBox. That would simplify the boxes. What do you think about this?